lib/src/ast/sass/statement/silent_comment.dart

```diff
@@ -12,11 +13,49 @@ class SilentComment implements Statement {
   /// The text of this comment, including comment characters.
   final String text;
 
+  /// The text of this comment that are marked as a documentation comment
+  /// (leading triple slashes).
+  ///
+  /// Provided with slashes and common leading whitespace removed.
+  String get docComment => _cleanup(text);
+
   final FileSpan span;
 
   SilentComment(this.text, this.span);
 
   T accept<T>(StatementVisitor<T> visitor) => visitor.visitSilentComment(this);
 
+  // Matches '///' followed by some amount of whitespace.
+  final _leadingWhitespace = RegExp('^(\/\/\/[\t ]*)[^\t ]?');
+
+  // Returns a string formed from lines of this comment that begin with '///'
+  // trimmed to remove the slashes and common leading whitespace.
+  String _cleanup(String text) {
```

Comment:
I don't think there's much benefit from making this a separate function, rather than just making it the body of `docComment`.

Reply:
Oops yeah, that was remnants from a previous experiment.